test/parallel/test-http-pipeline-requests-connection-leak.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+require('../common');
+const Countdown = require('../common/countdown');
+
+// This test ensures Node.js doesn't behave erratically when recieving pipelined
```

Comment:
Nit: receiving

Reply:
I think it's time I installed `vscode-spell-check` 😛